### PR TITLE
Add renovate config with Minikube updates

### PR DESCRIPTION
**Pull request by macaptain**
Add a renovate configuration which will automatically raise pull
requests for minikube updates.

Renovate pull requests will be automatically assigned to two of the
members of the OWNERS file who support the CI for this project. We could
automate the assignees list by using a CODEOWNERS file in future.

If this change is successful, we should see Renovate try to bump
minikube to version v1.21.0.

### CODE REVIEW

File: renovate.json
Comment (kashifest) [inline renovate.json:25]: We need to add everyone in the team may be?

**Review by kashifest — APPROVED** [review]

File: renovate.json
Comment (macaptain) [reply renovate.json:25]: What I've done here is the intersection of reviewers / approvers in the OWNERS file with our team. I could add all the members of our team regardless of whether they have permissions in this repo to approve or lgtm the changes?

Comment (kashifest) [reply renovate.json:25]: Make sense, but irrespective of approve/lgtm rights, reviewer can anyhow review the code was point

**Review by lentzi90 — APPROVED** [review]

**Review by namnx228 — COMMENTED** [review]
LGTM

File: renovate.json
Comment (namnx228) [reply renovate.json:25]: @macaptain I think you can add `reviewers` option to add team members who don't have right to approve or lgtm. Using this approach allows adding a person who has the right, and a person who may not have the privilege. 
https://docs.renovatebot.com/configuration-options/#reviewers

Comment (macaptain) [reply renovate.json:25]: @namnx228 that looks like a good solution. Since I have to add more dependencies to Renovate, I will add `reviewers` in the next update.

**Review by furkatgofurov7 — COMMENTED** [review]
This lgtm, looking forward for the first PR in this repo!

/lgtm

**Pull request merged**
